src/components/Modal.js

```diff
@@ -27,19 +29,26 @@ const styles = {
   }
 }
 
-/**
- * @param onDismiss
- * @param title
- * @param titleExtras
- * @param children
- * @param width=450
- * @param showCancel=true
- * @param showX=false
- * @param okButton
- */
 export default class Modal extends Component {
+  static defaultProps = {
+    showCancel: true,
+    showX: false,
+    width: 450
+  }
+
+  static propTypes = {
+    children: PropTypes.node,
+    okButton: PropTypes.oneOfType([PropTypes.string, PropTypes.func, PropTypes.any]),
```

Comment:
Should the last thing be PropTypes.node?

Reply:
Yup.